.yarnrc

```diff
@@ -6,4 +6,4 @@ lastUpdateCheck 1496303126136
 yarn-offline-mirror "./npm-packages-offline-cache"
 yarn-offline-mirror-pruning true
 --install.pure-lockfile true
---install.offline true
+#--install.offline true
```

Comment:
I don't think we want this permanently commented.

I've got the image building by doing the follow:

.yarnrc
```
#--install.pure-lockfile true
#--install.offline true
```

`yarn cache clean`
`yarn add cypress@4.6.0`

.yarnrc
```
--install.pure-lockfile true
--install.offline true
```

`docker build -t cypress-bs-01 -f cypress/Dockerfile .`
`docker run -it cypress-bs-01`

Able to run the Cypress Tools tests after setting the appropriate ENV vars.

Reply:
Updated